mobile_wallet/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.18.0
```

Comment:
Can you update the version in Cargo.toml & Cargo.lock to match this.

Then merge.